src/System.Management.Automation/engine/GetCommandCommand.cs

```diff
@@ -681,13 +679,7 @@ private bool IsNounVerbMatch(CommandInfo command)
         /// </summary>
         private void AccumulateMatchingCommands()
         {
-            Collection<string> commandNames =
-                SessionStateUtilities.ConvertArrayToCollection<string>(this.Name);
-
-            if (commandNames.Count == 0)
-            {
-                commandNames.Add("*");
-            }
+            var commandNames = Name != null && Name.Length != 0 ? Name : new string[] { "*" };
```

Comment:
```suggestion
            var commandNames = (Name != null && Name.Length != 0) ? Name : new string[] { "*" };
```

This seems like it reads a little bit clearer? Parens not technically needed here, but without them it becomes a bit tricky to read.